File: 3ra parte/flutter/start/lib/model/temp_model.dart

import 'package:json_annotation/json_annotation.dart';

part 'temp_model.g.dart';

@JsonSerializable()
class TempModel {
  double temp;
  @JsonKey(name: 'temp_min')
  double tempMin;
  @JsonKey(name: 'temp_max')
  double tempMax;

  TempModel(this.temp, this.tempMin, this.tempMax);

}

File: 3ra parte/swift/swift-start/TallerUNAJWeatherApp/Model/TempModel.swift
//
//  TempModel.swift
//  TallerUNAJWeatherApp
//
//  Created by itsupport on 16/08/2020.
//  Copyright © 2020 UNAJ. All rights reserved.
//

import Foundation

struct TempModel {
    let temp: Double
    let tempMin: Double
    let tempMax: Double

    init(temp: Double, tempMin: Double, tempMax: Double) {
        self.temp = temp
        self.tempMin = tempMin
        self.tempMax = tempMax
    }
}

/*
"main": {
  "temp": 281.52,
  "feels_like": 278.99,
  "temp_min": 280.15,
  "temp_max": 283.71,
  "pressure": 1016,
  "humidity": 93
},
*/
